pkg/cluster/install.go

```diff
@@ -93,6 +94,18 @@ func (m *manager) adminUpdate() []steps.Step {
 		)
 	}
 
+	if isRenewCerts {
+		toRun = append(toRun,
+			steps.Action(m.fixMCSCert),
+			steps.Action(m.configureAPIServerCertificate),
+			steps.Action(m.configureIngressCertificate),
+			steps.Action(m.initializeOperatorDeployer), // depends on kube clients
```

Comment:
Do we need this for cert renewals? I don't see anything that actually uses this client, but maybe I'm missing something.

Reply:
The function renewMDSDCertificate is dependant on initializeOperatorDeployer which will initialize the Operator.